packages/app/src/cli/models/app/loader.ts

```diff
@@ -414,7 +422,14 @@ class AppLoader {
       }
     })
 
-    return {allExtensions, usedCustomLayout: extensionDirectories !== undefined}
+    // Load app config modules
+    const appConfigModules = await loadAppConfigModules(
+      appDirectory,
+      this.specifications,
+      this.createExtensionInstance.bind(this),
+    )
```

Comment:
Can `loadConfigModules` be a method of the `loader` class so that you don't need to bind the create method or pass the specs?

Reply:
ooh good idea - i'll make the change